x-pack/plugins/infra/server/lib/adapters/framework/kibana_framework_adapter.ts

```diff
@@ -176,6 +176,6 @@ export class KibanaFramework {
       panels: [model],
       filters,
     };
-    return getVisData(requestContext, rawRequest, options);
+    return getVisData(requestContext, rawRequest, options) as Promise<InfraTSVBResponse>;
```

Comment:
could you please explain why it wass changed?

Reply:
@DziyanaDzeraviankina could you please try to export right types from TSVB plugin and replace `InfraTSVBResponse` to it? I see in `infra` plugin some types which can be probably safely removed
![image](https://user-images.githubusercontent.com/20072247/108961635-ade49280-7688-11eb-8bb9-0ca1cea6bab2.png)